### PR TITLE
packaging: release version 2.58

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

File: packaging/ubuntu-16.04/changelog [hunk 1]
```diff
@@ -1,7 +1,337 @@
...
+    - boot: add directory for data on install
+    - o/devicestate: change some names from esp to seed/seed-null
+    - gadget: add system-seed-null role
+    - o/devicestate: really add error to new error message
+    - restart,snapstate: implement reboot-required notifications on
+      classic
+    - many: avoid automatic system restarts on classic through new
+      overlord/restart logic
+    - release: Fix WSL detection in LXD
+    - o/state: introduce WaitStatusaction but there are two issues with
```
Comment (pedronis) [inline packaging/ubuntu-16.04/changelog:35]: this loosk broken

File: packaging/ubuntu-16.04/changelog [hunk 2]
```diff
@@ -1,7 +1,337 @@
...
+    - many: add stub services for prompting
+    - tests: add libfwupd and libfwupdplugin5 to openSUSE dependencies
+    - o/snapstate: fix snaps-hold pruning/reset in the presence of
+      system holding
+    - many: add support for setting up encryption from installer
+    - many: support classic snaps in the context of classic and extended
+      models
+    - o/auth,o/devicestate: review feedback
+    - daemon,tests: review feedback
+    - client,cmd/snap,daemon: review feedback
```
Comment (pedronis) [inline packaging/ubuntu-16.04/changelog:119]: not sure what to do about this, but this kind of entries are really low value

File: packaging/ubuntu-16.04/changelog [hunk 3]
```diff
@@ -1,7 +1,337 @@
...
+    - daemon: add `ensureStateSoon()` when calling systems POST api
+    - gadget: add rules for validating classic with modes gadget.yaml
+      files
+    - wrappers: journal namespaces did not honor journal.persistent
+    - many: stub devicestate.Install{Finish,SetupStorageEncryption}()
+    - sandbox/cgroup: don't check V1 cgroup if V2 is active
+    - seed: add support to load auto import assertion
+    - tests: fix preseed tests for arm systems
+    - include/lk: update LK recovery environment definition to include
+      devi…
```
Comment (pedronis) [inline packaging/ubuntu-16.04/changelog:214]: where is ... coming from?

**Review by pedronis — COMMENTED** [review]
some comments

File: packaging/ubuntu-16.04/changelog (hunk 3 above)
Comment (mvo5) [reply packaging/ubuntu-16.04/changelog:214]: Meh, this was merged like this sadly (https://github.com/snapcore/snapd/pull/11128) :(

**Review by pedronis — APPROVED** [review]
thanks

**Pull request merged**